### PR TITLE
chore: add change channel e2e test

### PR DESCRIPTION
**Pull request by pandemicsyn**
#### What this PR does / why we need it:

Add a "Change channel" e2e playwright test that:

- Associates customer with only the "automated" channel
- Install's 1.0.0 from the "automated" channel
- Updates the customers channels from "automated" to "alternate" channel
- Sync's the license
- Deploys 1.0.3 from the "alternate" channel

This is a needed to test/ensure that license channel changes are still supported as they are today in #4767 

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: e2e/playwright/tests/change-channel/test.spec.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    await page.getByText('Sync license').click();
+
+    await expect(page.getByLabel('Next step')).toContainText('License synced', { timeout: 10000 });
+    await page.getByRole('button', { name: 'Ok, got it!' }).click();
+
+    await expect(page.locator('#app')).toContainText('Alternate');
+    await expect(page.locator('#app')).toContainText('1.0.3', { timeout: 10000 });
+    await expect(page.locator('#app')).toContainText('Upstream Update', { timeout: 10000 });
+
+    await page.waitForTimeout(5000);
```
Comment (sgalsaleh) [inline e2e/playwright/tests/change-channel/test.spec.ts:27]: why is this needed?

Comment (pandemicsyn) [reply e2e/playwright/tests/change-channel/test.spec.ts:27]: as sorry , this may actually not be needed anymore since just removed the preflight checks from the alternate channel. Originally was using this to try and give the preflight checks a chance to complete. Let me remove it and see if theres remaining timing issues.
